cmd/anonymizer/app/query/empty_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package query
```

Comment:
Let's remove this file. You already have .nocover, but I think adding empty test file may still force the zero coverage to be added and impact the total ratio. Right now your PR does not pass the code coverage threshold because it drags the project-level measure below 95%.